app/forms/routing/rate_group_duplicator_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Routing::RateGroupDuplicatorForm < ApplicationForm
+  attribute :name, :string
+  attribute :id, :integer
+
+  validates :name, :id, presence: true
+
+  validate do
+    errors.add(:id, :invalid) unless Routing::RateGroup.exists?(id)
+    errors.add(:name, :taken) if Routing::RateGroup.exists?(name: name)
+  end
+
+  # Required by activeadmin https://github.com/activeadmin/activeadmin/pull/5253#discussion_r155525109
+  def self.inheritance_column
+    nil
+  end
```

Comment:
I think this method already defined in `ApplicationForm`

if not - move it to  `ApplicationForm`

Reply:
and remove this method from all `ApplicationForm` subclasses